editor/plugins/org.fusesource.ide.camel.editor/src/org/fusesource/ide/camel/editor/properties/DetailsSection.java

```diff
@@ -128,6 +128,23 @@ protected void generateTabContents(List<Parameter> props, final Composite page,
             // DESCRIPTION PROPERTIES
             if (CamelComponentUtils.isDescriptionProperty(prop)) {
 				new DescriptionParameterPropertyUICreator(dbc, modelMap, eip, selectedEP, p, page, getWidgetFactory()).create();
+            } else if ("redeliveryPolicy".equals(prop.getName())) {
```

Comment:
can you put back this code at the initial place to avoid any surprise and have less modified code for an easier review, please?